site/en/docs/extensions/mv3/promises/index.md

```diff
@@ -171,18 +172,21 @@ chrome.tabs.create({...})
   });
 ```
 
-Instead of setting `chrome.runtime.lastError`, we deliver the error directly to the "catch" of the
-promise.
+Extensions APIs don't set `chrome.runtime.lastError` when you use a promise, instead returning the
```

Comment:
Is "returning" fine here grammatically? It reads to me like it should be something like "they return".

Reply:
It's OK grammatically but not optimal (thinking non-native speakers, localization, etc.)

Rephrased.